### PR TITLE
Banana oatmeal cookies

### PR DESCRIPTION
**Pull request by SoyaBeanie**
Adding recipe for banana oatmeal cookies

### CODE REVIEW

File: src/recipes/banana-oatmeal-cookies.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+---
+tags: ["2 Ingredient", "cookies"]
```
Comment (jlooper) [inline src/recipes/banana-oatmeal-cookies.md:3]: hi, can you categorize this just as 'Dessert' please, to keep the tags simple :)

File: src/recipes/banana-oatmeal-cookies.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+title: "banana oatmeal cookies"
```
Comment (jlooper) [inline src/recipes/banana-oatmeal-cookies.md:4]: hi, please capitalize each letter of the recipe title

**Review by jlooper — COMMENTED** [review]
just a few small changes

**Comment by jlooper** [comment]
@SoyaBeanie just a few small edits

File: src/recipes/banana-oatmeal-cookies.md (hunk 1 above)
Comment (SoyaBeanie) [reply src/recipes/banana-oatmeal-cookies.md:3]: Dessert instead of cookies? Still include '2 Ingredient' ?

**Review by jlooper — APPROVED** [review]

**Pull request merged**